README.md

```diff
@@ -2,21 +2,30 @@
 
 This repository contains the completed app and starter files for the [Build a Shopify app with Node and React](#) tutorial.
```

Comment:
here too (#) (https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react)